lang/cs/Org.Apache.REEF.Utilities/Org.Apache.Reef.Utilities.DotNet.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Org.Apache.Reef.Utilities</AssemblyName>
+    <OutputPath>bin\</OutputPath>
+    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard2.0' ">2.0.0-beta-25123-01</NetStandardImplicitPackageVersion>
+    <DefaultItemExcludes>$(DefaultItemExcludes);**/AssemblyInfo.cs</DefaultItemExcludes>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.2-beta2" />
+  </ItemGroup>
+  
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
```

Comment:
This needs to be .NET 4.5.1 with Saikat's recent changes and need to add .NET 4.6.